snippets/tomorrow.md

```diff
@@ -1,10 +1,20 @@
 ### tomorrow
 
 Results in a string representation of tomorrow's date.
-Use `new Date()` to get today's date, adding `86400000` of seconds to it(24 hours), using `Date.toISOString()` to convert Date object to string.
+Use `new Date()` to get today's date, adding one day using `Date.getDate()` and `Date.setDate()`, and converting the Date object to a string.
 
 ```js
-const tomorrow = () => new Date(new Date().getTime() + 86400000).toISOString().split('T')[0];
+const tomorrow = () => {
+  let t = new Date();
+  t.setDate(t.getDate() + 1);
+  return (
+    t.getFullYear() +
+    '-' +
+    String(t.getMonth() + 1).padStart(2, '0') +
+    '-' +
+    String(t.getDay()).padStart(2, '0')
```

Comment:
`getDay` is 0-6(M-F), `getDate` is 0-End of Month. If we're meaning to emulate the ISOString you need to use `t.getDate()`